### DISCUSSION TITLE
Open Source Variant of Nova-2

### DISCUSSION THREAD

**Discussion by sanchit-gandhi** [deepgram/community#351] [Feature Requests]
Hey Deepgram team 👋

Wondering if you had any plans to open-source Nova-2 (or a variant thereof)? We’ve seen from OpenAI and Whisper the effectiveness of open-sourcing part of a model to generate interest and the long-term usage that comes off the back of it. The community is super interested in running performant ASR models fast. Here, Nova-2 has a big advantage over Whisper and could really take over for English ASR!

We could integrate the model directly into Hugging Face Transformers 🤗 and do a joint announcement to maximise visibility + usability of an open-source release. This would also reduce the burden of a code release on your behalf since it could all live in transformers from the get-go.

Let me know if this is something that you’d be keen for!

**Comment by lukeocodes** [comment]
Hey Sanchit, thanks! Let me ask around a bit about this. It's something I am keen to help you with, but I can't say yes or no right now.

**Comment by jpvajda** [comment]
@sanchit-gandhi  Thanks for reaching out about this collaboration I'll help @lukeocodes  get some answers to your question as well. Looking forward to discussing more soon.

**Reply by jpvajda** [reply]
@sanchit-gandhi  I wanted to try to connect with you on what you proposed in this discussion to learn more. What is the best way to reach you? If you want to send your contact email to `devrel-at-deepram-dotcom` we can try to connect. Thank you!

**Comment by jpvajda** [comment]
@sanchit-gandhi  I just wanted to let you know the Deepgram DX Team setup a [Deepgram Org](https://huggingface.co/Deepgram) on Hugging Face. 

We haven't yet decided which Deepgram model to open source but when we do we can reach out. I have no timeline for when we'd do this but I wanted to let you know we are exploring how to be more visible on 🤗 .
